microproxy/protocol/tls.py

```diff
@@ -74,16 +74,13 @@ def create_dest_sslcontext(insecure=False, trusted_ca_certs="", alpn=None):
     ssl_ctx = create_basic_sslcontext()
 
     if not insecure:
+        verify_mode = SSL.VERIFY_PEER | SSL.VERIFY_FAIL_IF_NO_PEER_CERT
         trusted_ca_certs = trusted_ca_certs or certifi.where()
         ssl_ctx.load_verify_locations(trusted_ca_certs)
-        ssl_ctx.set_verify(
-            SSL.VERIFY_PEER | SSL.VERIFY_FAIL_IF_NO_PEER_CERT,
-            certificate_verify_cb)
     else:
-        def verify_cb_always_true(*args, **kwargs):
-            return True
-        ssl_ctx.set_verify(SSL.VERIFY_NONE, verify_cb_always_true)
+        verify_mode = SSL.VERIFY_NONE
 
+    ssl_ctx.set_verify(verify_mode, certificate_verify_cb)
```

Comment:
I'm not sure that the way is better.
It seems more complicated that I need to read to the 83rd line to realize that the verify_mode is used to set the context verify mode.
Not very straightforward for me.
How do you think?